modules/ob-deleting-a-build-resource.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: PROCEDURE
+[id="ob-deleting-a-build-resources_{context}"]
+= Deleting a build resource
+
+You can delete a `build` resource if it is not required in your project.
+
+.Prerequisites
+
+* You have installed the `oc` CLI.
+* Optional: You have installed the shp CLI.
+
+.Procedure
+
+. Delete a `build` resource by using one of the CLIs:
```

Comment:
A single step cannot be part of a numbered list. So, we can convert it into a bullet list. The subsequnet commands into sub-bullets.
Applicable to all delete-related procedures and any other procedures with a single step.

Reply:
Also above this sectionhttps://99107--ocpdocs-pr.netlify.app/openshift-builds/latest/work_with_builds/using-builds#uploading-the-OCI-artifact-by-using-the-CLI_using-builds, there are two methods available from which the user can choose one. So, no need to put them in numbered list (a,b), convert them into bullet lists because there are no sequence between the two methods. Users might perceive that they have to perform both the methods in a sequence, which is not correct.

Reply:
In [this section](https://99107--ocpdocs-pr.netlify.app/openshift-builds/latest/work_with_builds/using-builds#ob-creating-a-buildpacks-build_using-builds), we have oc and shp cli command examples in a numbered sequence (a, b), we need to conver them into a bullet list, as the users need not to run those commands in sequence, but they can run any of the commands based on the CLI they want to use.